docs/whats-new/dotnet-docs-mod2.md

```diff
@@ -1,108 +1,137 @@
-ms.custom: February-2023
-ms.date: 03/01/2023
+title: ".NET docs: What's new for May 2023"
+description: "What's new in the .NET docs for May 2023."
+ms.custom: May-2023
+ms.date: 06/01/2023
 ---
 
-# .NET docs: What's new for February 2023
+# .NET docs: What's new for May 2023
 
-Welcome to what's new in the .NET docs for February 2023. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the .NET docs for May 2023. This article lists some of the major changes to docs during this period.
 
 ## .NET breaking changes
 
 ### New articles
 
-- [Solution-level `--output` option no longer valid for build-related commands](../core/compatibility/sdk/7.0/solution-level-output-no-longer-valid.md)
-- ['dotnet pack' uses Release configuration](../core/compatibility/sdk/8.0/dotnet-pack-config.md)
-- ['dotnet publish' uses Release configuration](../core/compatibility/sdk/8.0/dotnet-publish-config.md)
-- [DateTime addition methods precision change](../core/compatibility/core-libraries/7.0/datetime-add-precision.md)
-- [ExceptionCollection ctor throws ArgumentException](../core/compatibility/windows-forms/8.0/exceptioncollection.md)
-- [ImageList.ColorDepth default is Depth32Bit](../core/compatibility/windows-forms/8.0/imagelist-colordepth.md)
-- [TableLayoutStyleCollection throws ArgumentException](../core/compatibility/windows-forms/8.0/tablelayoutstylecollection.md)
-- [Forms scale according to AutoScaleMode](../core/compatibility/windows-forms/8.0/top-level-window-scaling.md)
-- [ITypeDescriptorContext nullable annotations](../core/compatibility/core-libraries/8.0/itypedescriptorcontext-props.md)
+- [Rate-limiting middleware requires AddRateLimiter](../core/compatibility/aspnet-core/8.0/addratelimiter-requirement.md)
+- [ISystemClock is obsolete](../core/compatibility/aspnet-core/8.0/isystemclock-obsolete.md)
+- [System.Drawing.Common config switch removed](../core/compatibility/core-libraries/7.0/system-drawing.md)
+- [Date and time converters honor culture argument](../core/compatibility/globalization/8.0/typeconverter-cultureinfo.md)
+- [ConcurrencyLimiterMiddleware is obsolete](../core/compatibility/aspnet-core/8.0/concurrencylimitermiddleware-obsolete.md)
+- [Custom converters for serialization removed](../core/compatibility/aspnet-core/8.0/problemdetails-custom-converters.md)
+- [StripSymbols defaults to true](../core/compatibility/deployment/8.0/stripsymbols-default.md)
+- [GC.GetGeneration might return Int32.MaxValue](../core/compatibility/core-libraries/8.0/getgeneration-return-value.md)
+- [API obsoletions with non-default diagnostic IDs (.NET 8)](../core/compatibility/core-libraries/8.0/obsolete-apis-with-custom-diagnostics.md)
+- [BinaryFormatter disabled across most project types](../core/compatibility/serialization/8.0/binaryformatter-disabled.md)
+
+### Updated articles
+
+- [Breaking changes in .NET 7](../core/compatibility/7.0.md) - System.Drawing.Common config switch removal
```

Comment:
Let's add it. (I think the reason is because there were several PRs, but no single PR was very big:

```suggestion
- [Breaking changes in .NET 8](../core/compatibility/8.0.md)
   - Add EF Core breaking changes
   - Document Preview 5 breaking changes for ASP.NET
   - BinaryFormatter disabled & obsoletion of legacy serialization APIs
   - StripSymbols default for PublishAOT
   - ConvertTo breaking change
   - add getgeneration breaking change
   - Two ASP.NET breaking changes for .NET 8
- [Breaking changes in .NET 7](../core/compatibility/7.0.md) - System.Drawing.Common config switch removal
```